docs/plugins/tomcat.md

```diff
@@ -0,0 +1,44 @@
+# `tomcat` plugin
+
+The `tomcat` plugin consumes [Apache Tomcat](https://tomcat.apache.org/) log entries from the local filesystem and outputs parsed entries.
+
+## Configuration Fields
+
+| Field | Default | Description |
+| --- | --- | --- |
+| `source` | `file` | Use this field to specify where your logs are coming from. When choosing the 'file' option, the agent reads in logs from the log paths specified below. When choosing the 'Kubernetes' options, the agent reads logs from /var/log/containers based on the Pod and Container specified below. |
+| `log_format` | `default`  | When choosing the 'default' option, the agent will expect and parse logs in a format that matches the default logging configuration. When choosing the 'observIQ' option, the agent will expect and parse logs in an optimized JSON format that adheres to the observIQ specification, requiring an update to the server.xml file. |
+| `cluster_name` | `""` | Cluster Name to be added to a resource label |
+| `pod_name` | `'tomcat-*'` | The pod name (without the unique identifier on the end) |
+| `container_name` | `"*"` | The container name of the Nginx container |
```

Comment:
See mongodb feedback, it applies here as well.

Reply:
updated similar to mongodb